meeting-minutes/2024/2024-09-18.md

```diff
@@ -0,0 +1,128 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+- Justin & Omar open the meeting.
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 8 or more of the 14 voting members (including the officers).
+
+| First Name | Last Name   | Company                                                     | Role(s)       | Present |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|:--------|
+| Adrian     | Diglio      | Microsoft                                                   | Member        | No      |
+| Art        | Middlekauff | Flexera                                                     | Voting Member | No      |
+| David      | Kemp        | National Security Agency                                    | Member        | No      |
+| Denny      | Page        | Individual                                                  | Voting Member | Yes     |
+| Feng       | Cao         | Oracle                                                      | Voting Member | Yes     |
+| Harin      | Sarda       | Cisco Systems                                               | Voting Member | Yes     |
+| Jautau     | White       | Microsoft                                                   | Voting Member | No      |
+| Jeremy     | Rickard     | Microsoft                                                   | Member        | No      |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair      | Yes     |
+| Kris       | Vandecruys  | Cisco Systems                                               | Member        | Yes     |
+| Kunal      | Modasiya    | Qualys, Inc.                                                | Member        | No      |
+| Langley    | Rock        | Dell                                                        | Voting Member | Yes     |
+| Martin     | Prpic       | Red Hat                                                     | Member        | No      |
```

Comment:
Tracking that Martin had voting status at this time. Should have lost Voting status after October meeting.
```suggestion
| Martin     | Prpic       | Red Hat                                                     | Voting Member        | No      |
```